Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "starship"
-version = "1.4.0"
+version = "1.59"
```

Comment:
This sets the version of starship and not the supported rust version. Please also add a comment that the MSRV is intended as a hint and that only the latest version is tested, and that the specified minimum rust version is not guaranteed to work.

Reply:
Hi @davidkna Can you please the exact location of the rust version where it resides so I can change it.

Reply:
There is no minimum rust version specified in the file. You will need to add it.

Reply:
In which file?

Reply:
https://github.com/starship/starship/issues/3716 includes a link to the relevant documentation.

Reply:
Hi @davidkna I have added the rust version can you please review it.

Reply:
Please ensure that `Cargo.toml` is still valid TOML, ensure the formatting (dprint, see the contributing guidelines) passes and add the comment I requested.